util/http_utils.go

```diff
@@ -78,7 +78,7 @@ func GetServerEnvVars() (readTimeOutInt, writeTimeOutInt int, addr, err string)
 		return -1, -1, "", atoiErr.Error()
 	}
 
-	return readTimeOutInt, writeTimeOutInt, adrr, ""
+	return readTimeOutInt, writeTimeOutInt, addr, ""
```

Comment:
This typo fix is not related to the PR title. It is better to make a separated PR for the typo.

Reply:
so should I revert the commit and open a new PR?

Reply:
added as a comment